### ISSUE TITLE
Update to latest rococo-v1

### ISSUE THREAD

**Issue by riusricardo** [substrate-developer-hub/cumulus-workshop#60]
The Cumulus workshop contains Polkadot chain specs that need to be updated to the specified commits at the same time as the docs.

**Comment by riusricardo** [comment]
@NukeManDan  Also take into account that for clarity the changes need to update the commit in the compilation page. Keep the commit as part of the instruction and do not direct the user to look for the commits on the README.

**Comment by NukeManDan** [comment]
So sorry about the misstep on the included files! I have #62 open to get things properly addressed. 

I am not sure what you mean about the README. Possibly the link to the "home" page ("Preparation" in the sidebar / "Substrate Cumulus Workshop" as the first h1) of the workshop when rendered? That is the only link to a README that is in the present site. 

I have amended that to a simplified link in #62

**Comment by riusricardo** [comment]
> So sorry about the misstep on the included files! I have #62 open to get things properly addressed. 
> 
> I am not sure what you mean about the README. Possibly the link to the "home" page ("Preparation" in the sidebar / "Substrate Cumulus Workshop" as the first h1) of the workshop when rendered? That is the only link to a README that is in the present site. 
> 
> I have amended that to a simplified link in #62 

No worries, it's just a heads up. I'm talking about this:
```
# Switch into the Polkadot directory cd polkadot # Checkout the proper commit git checkout 943038a
```
I added back the commit instead of the latest change where it asked the user to find out from the README which commit should use.
